### ISSUE TITLE
Translate privacy policy in french

### ISSUE THREAD

**Issue by ludoo0d0a** [S-Man42/GCWizard#446]
translate privacy policy in french and adapt url

**Comment by S-Man42** [comment]
Well, if you want to. But that's only for US laws (requested by Google and Apple). But it is not required in other languages. There's not even a German version ;) For German laws (because website is hosted in Germany) theres a German "Impressum". I think I dont need such things for any other languages...

**Comment by ludoo0d0a** [comment]
for GDPR, I think it should be there...
I can ask to a friend, DPO

**Comment by S-Man42** [comment]
No. From my point of view, it's not necessary. The polocy was only for Google and Apple, because it is demanded by US Law. There's also no German version of it at the moment.
